src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -468,10 +394,10 @@ def pool_acl_verification(self, current_user_acl, read, write,
         # (6)Verify pool acl ace update and delete
         self.log.info("  (6)Verify update and delete of pool's acl entry.")
         tmp_ace = "daos_ci_test_new"
-        new_entrys = [acl_entry("user", tmp_ace, ""),
-                      acl_entry("group", tmp_ace, "")]
-        acl_principals = [acl_principal("user", tmp_ace),
-                          acl_principal("group", tmp_ace)]
+        new_entrys = [secTestBase.acl_entry("user", tmp_ace, "", PERMISSIONS),
```

Comment:
(style)  'entrys' may be misspelled - perhaps 'entries'?

Reply:
Done